frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/components/DesktopInbox.js

```
@@ -41,35 +45,83 @@ const DesktopInbox = ({ tableConfig, filterComponent, ...props }) => {
+                <td style={{ ...tdStyle, ...statusStyle }}>
+                  {property.searchData.status === 'INWORKFLOW' ? 'Correction Pending' : property.searchData.status}
+                </td>
+                <td style={tdStyle}>{property.searchData.propertyId}</td>
+                <td style={tdStyle}>
+                  <div style={buttonContainerStyle}>
+                    <button style={buttonStyle} onClick={() =>
+                      history.push({
+                        pathname: `/digit-ui/employee/pt/PTPropertyTaxForm/${property.searchData.propertyId}`,
+                        state: { propertyData: property }
+                      })
+                    }>Edit Property Address</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)